e2e/scenario_helpers_test.go

```diff
@@ -122,9 +129,8 @@ func createAndValidateVM(ctx context.Context, t *testing.T, scenario *Scenario)
 	t.Logf("node %s is ready, proceeding with validation commands...", vmssName)
 
 	vmPrivateIP, err := getVMPrivateIPAddress(ctx, *scenario.Runtime.Cluster.Model.Properties.NodeResourceGroup, vmssName)
-	require.NoError(t, err)
```

Comment:
we are ok with the `err` should we print it ?

Reply:
There we a duplicate check since we call `require.NoError` again below so I removed this one. But if the `err` is not nil, `require.NoError` wil log it.